testsuite/do_tests.sh.in

```diff
@@ -730,7 +730,20 @@ if test "x${TEST_PYNEST}" = xtrue ; then
         echo "  Using nosetests."
         echo
 
-        nosetests -v --with-xunit --xunit-file=${TEST_OUTDIR}/pynest_tests.xml ${NEST_PYTHON_PREFIX:-@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@}/nest/tests ${NEST_PYTHON_PREFIX:-@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@}/nest/topology/tests 2>&1 \
+        # if possible, pick the python version that was specified during the installation
+
+        WITH_PYTHON=`cat CMakeCache.txt | grep with-python | cut -d "=" -f2`
```

Comment:
Isn't the Python executable used for configuration available in CMake's variable `PYTHON` and couln't that be just replaced into this file using `@PYTHON@`?